tensorboard/webapp/runs/store/runs_selectors_test.ts

```diff
@@ -415,13 +411,9 @@ describe('runs_selectors', () => {
     it('returns groupBy set by user when it is present', () => {
       const state = buildStateFromRunsState(
         buildRunsState({
-          userSetGroupBy: {
-            key: GroupByKey.REGEX,
-            regexString: 'hello',
-          },
-          initialGroupBy: {
-            key: GroupByKey.RUN,
-          },
+          colorGroupRegexString: 'hello',
+          initialGroupBy: {key: GroupByKey.RUN},
+          userSetGroupByKey: GroupByKey.REGEX,
```

Comment:
Could you add a case for:
```
colorGroupRegexString: '',
initialGroupBy: {key: GroupByKey.REGEX, regexString: 'hi'},
userSetGroupByKey: GroupByKey.REGEX,
```

I honestly do not know what the right behavior should be but we should codify our behavior in this test spec.

Reply:
thanks for pointing this out. I think this means the user set groupBykey with '' (empty) string so the the result is 
`{key: GroupByKey.REGEX, regexString: ''}`